internal/cloud/atlas/groups.go

```diff
@@ -17,15 +17,25 @@ type Group struct {
 	Name string `json:"name"`
 }
 
+type link struct {
+	Href string `json:"href"`
+	Rel  string `json:"rel"`
+}
+
 type groupResponse struct {
 	Results []Group `json:"results"`
+	Links   []*link `json:"links"`
 }
 
 func (c *client) Groups() ([]Group, error) {
-	res, resErr := c.do(
+	return c.FetchGroups(c.baseURL+GroupsPath, api.RequestOptions{})
```

Comment:
(for testing purposes) broke this out into its own func instead of keeping as a nested func